root/edit/info.tt

```diff
@@ -1,7 +1,7 @@
 [% IF summary %]
     [% IF edit.status != 1 && edit.status != 2 %]
         <div class="edit-status">
-            [%- edit.status_name -%]
+            [%- l(edit.status_name) -%]
```

Comment:
Seems good then.

(Hmm, I thought you could switch the status to “recommend merge” or something like that now?)